dubbo-samples-merge/dubbo-samples-merge-consumer/src/main/java/org/apache/dubbo/samples/merge/DubboConsumer.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.dubbo.samples.merge;
+
+import org.apache.dubbo.samples.merge.api.IDubboService;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class DubboConsumer {
+    public static void main(String[] args) throws Exception {
+        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("spring/dubbo-consumer.xml");
+        context.start();
+
+        IDubboService dubboService = (IDubboService) context.getBean("dubboService");
```

Comment:
Pls convert this case to integration test so it can verify automatically

Reply:
yeah, a file which was commited a few minutes ago.